modules/client/src/client-common/interfaces/plugin.ts

```diff
@@ -67,6 +72,11 @@ export interface ICanVoteParams {
   address: string;
 }
 
+export type CanExecuteParams = {
+  proposalId: string;
+  addressOrEns: string;
```

Comment:
As said, is this needed?

Reply:
same, its in the contracts

Reply:
Not sure where you see it

https://github.com/aragon/core/pull/167/files#diff-aa6435e11ba402b1ca24319f8bfef661e834e2124a93b4d826586bd8ce2e64a4R286